pkgs/development/python-modules/s3transfer/default.nix

```diff
@@ -1,13 +1,5 @@
-{ lib
-, botocore
-, buildPythonPackage
-, docutils
-, fetchFromGitHub
-, mock
-, pytestCheckHook
-, pythonOlder
-, wheel
-}:
+{ lib, botocore, buildPythonPackage, docutils, fetchFromGitHub, mock
+, pytestCheckHook, pythonOlder, stdenv, wheel }:
```

Comment:
Why rearrange the args..? I think the one-per-line form is generally favoured these days because it leads to fewer merge conflicts.

Reply:
I can undo that. I think `nixfmt` might've done that.